addins/Cake.SharePoint.yml

```diff
@@ -0,0 +1,9 @@
+Name: Cake.SharePoint
+NuGet: Cake.SharePoint
+Assemblies:
+- "/**/Cake.SharePoint.dll"
+Repository: https://github.com/orialmog/Cake.ProjHelpers/
```

Comment:
Repository URL doesn't seem to match. Should probably be fixed first upstream

Reply:
The `Project Site` link on this package's [Nuget page](https://www.nuget.org/packages/Cake.SharePoint/) indeed points to the wrong github project. The author needs to double check their nuspec.

Reply:
I was going to raise an issue, but I searched GitHub and can't find a project for this addin.

Reply:
Also had a quick look, but couldn't find it. Found a Bitbucket account for this user, but code also doesn't seem to be there.

Reply:
I have sent the user a message through NuGet.  Hopefully they will get back to us.

Reply:
Hi,
I am sorry about this!
I took the nuspec file from another Cake addon as an example and forgot to edit those fields...
I created a repo and updated the nuspec file. I uploaded a new 1.0.3 NuGet release that should have all the correct info

Reply:
Do I need to add certain files to the repo for the API documentation?

Reply:
@JeroenVandezande no need to apologise!  Thanks for getting in touch, and for making that change, I really appreciate it!

Reply:
Are any other changes needed?

Reply:
I didn't notice anything else that needs to be changed but I encourage you to regularly check the [audit report](https://github.com/cake-contrib/Home/blob/master/Audit.md) to see if our automated process has detected any best practice that your addin is not following.